### PR TITLE
admin-service-accounts-admin-pr-2017-08-06

### PR DESCRIPTION
**Pull request by zhangqx2010**
> NOTE: Please check the “Allow edits from maintainers” box (see image below) to  
> [allow reviewers to fix problems](https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/) on your patch and speed up the review process.
>
> Please delete this note before submitting the pull request.

Allow edits from maintainers checkbox

### CODE REVIEW

File: docs/admin/service-accounts-admin.md [hunk 1]
```diff
@@ -6,15 +6,21 @@ assignees:
 - liggitt
 title: Managing Service Accounts
```
Comment (markthink) [inline docs/admin/service-accounts-admin.md:7]: 所有标题未翻译..

Comment (zhangqx2010) [reply docs/admin/service-accounts-admin.md:7]: 好的，原本是担心标题会有位置标签被引用，所以没有翻译，看来是多虑了哈

**Comment by zhangqx2010** [comment]
updated, plz review @markthink

File: docs/admin/service-accounts-admin.md (hunk 1 above)
Comment (markthink) [reply docs/admin/service-accounts-admin.md:7]: 标题的中文可以直接把原标题替换，不需要放到小括号里

File: docs/admin/service-accounts-admin.md [hunk 2]
```diff
@@ -6,15 +6,24 @@ assignees:
...
+-->
+*本文是 service accounts 管理员手册。需要[Service Accounts 用户手册](/docs/user-guide/service-accounts)的知识背景。*
+
+*对于 authorization 和 user accounts 支持已经在计划中，但是还未完成。为了更好地解释 service
+accounts，有时会提到一些未完成的功能特性。*
 
+<!--
 ## User accounts vs service accounts
+-->
+## User accounts vs service accounts(用户账号 vs 服务账号)
```
Comment (markthink) [inline docs/admin/service-accounts-admin.md]: 英文已经注释过了，可以去掉英文内容
```
## 用户账号与服务账号比较
```

**Comment by markthink** [comment]
@zhangqx2010 标题的英文可以移除。

**Comment by zhangqx2010** [comment]
@markthink  done

**Comment by markthink** [comment]
@zhangqx2010 OK

**Pull request merged**